.circleci/config.yml

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+
 version: 2.1
 commands:
   install-bazel:
```

Comment:
Now that there is `install-bazel-mac`, let's rename this to `install-bazel-linux`, Ganesh.

Reply:
Fixed in 49716f26ca94d41f3d86c8bf6a4082512f23ca8b

Reply:
Fixed in 49716f26ca94d41f3d86c8bf6a4082512f23ca8b